hexa/workspaces/schema.py

```diff
@@ -109,4 +155,55 @@ def resolve_delete_workspace(_, info, **kwargs):
         return {"success": False, "errors": ["PERMISSION_DENIED"]}
 
 
-workspaces_bindables = [workspace_queries, workspace_object, worskspace_mutations]
+@workspace_mutations.field("inviteWorkspaceMember")
+def resolve_create_workspace_member(_, info, **kwargs):
+    request: HttpRequest = info.context["request"]
+    input = kwargs["input"]
+    try:
+        workspace: Workspace = Workspace.objects.filter_for_user(request.user).get(
+            id=input["workspaceId"]
+        )
+        user = User.objects.get(email=input["userEmail"])
+
+        workspace_membership = WorkspaceMembership.objects.create_if_has_perm(
+            principal=request.user, workspace=workspace, user=user, role=input["role"]
+        )
+        send_mail(
+            title=gettext_lazy("You've been added to the workspace '{workspace.name}'"),
+            template_name="workspaces/mails/invite_member",
+            template_variables={
+                "workspace": workspace.name,
+                "owner": request.user.display_name,
+                "workspace_url": "{url}/workspaces/{workspace_id}".format(
+                    url=settings.NEW_FRONTEND_DOMAIN, workspace_id=workspace.id
+                ),
+            },
+            recipient_list=[user.email],
+        )
+        return {
+            "success": True,
+            "errors": [],
+            "workspace_membership": workspace_membership,
+        }
+    except (Workspace.DoesNotExist, User.DoesNotExist):
```

Comment:
You have to split the two cases since it's not the same to not find the user as to not find the workspace you try to use.